src/content/subgames.cpp

```diff
@@ -67,15 +67,22 @@ SubgameSpec findSubgame(const std::string &id)
 	std::vector<GameFindPath> find_paths;
 	while (!search_paths.at_end()) {
 		std::string path = search_paths.next(PATH_DELIM);
-		find_paths.emplace_back(path + DIR_DELIM + id, false);
-		find_paths.emplace_back(path + DIR_DELIM + id + "_game", false);
+		path += DIR_DELIM;
```

Comment:
I prefer path.append(DIR_DELIM).append(id) to prevent multiple useless lines

Reply:
Tested, works, so here you are.